addons/website/static/src/snippets/s_dynamic_snippet/options.js

```diff
@@ -4,9 +4,18 @@ odoo.define('website.s_dynamic_snippet_options', function (require) {
 const options = require('web_editor.snippets.options');
 
 const dynamicSnippetOptions = options.Class.extend({
+    /**
+     * @typedef {Object} Template - definition of a dynamic snippet template
+     * @property {string} key - key of the template
+     * @property {string} numOfEl - number of elements on desktop
+     * @property {string} numOfElSm - number of elements on mobile
+     * @property {string} numOfElFetch - number of elements to fetch
+     * @property {string} rowPerSlide - number of rows per slide
+     * @property {string} arrowPosition - position of the arrows
+     * @property {string} extraClasses - classes to be added to the <section>
+     */
```

Comment:
What does this documents? :thinking:

Reply:
This documents the template infos that we get from the `/website/snippet/filter_templates` route. I then reuse it to define the `this.dynamicFilterTemplates` property.

I honestly hesitated to leave it, but thought that it might be worth it since it's not always clear what the route sends back to you.

Do you want me to remove it? Or add comments to make it clearer what it represents?